08week/star-wars-app/src/Heroes.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './Heroes.css'
+
+
+const Heroes = (props) => {
+  return (
+    <div
+      style={{
+        backgroundColor: props.type === 'light' ? 'white' : 'white',
+      }}
+    >
+      {props.heroes.length > 0 && props.heroes
```

Comment:
nice component, it's best practice to pull out this mapping function into an external function